tests/test_spark.py

```diff
@@ -175,6 +175,18 @@ def test_jars_flag(service_name=utils.SPARK_SERVICE_NAME):
         args=["--jars {}".format(uploadedJarUrl),
               "--class MultiConfs"])
 
+@pytest.mark.sanity
+def test_packages_flag(service_name=utils.SPARK_SERVICE_NAME):
+    utils.run_tests(
+        app_url=utils.dcos_test_jar_url(),
+        app_args="20",
+        expected_output="The sum of 20 numbers is 210",
```

Comment:
Let's change this to a number-only check.

Reply:
Changed it to the number only check.